src/Groups/directproducts.jl

```diff
@@ -504,13 +530,18 @@ W(g_1,...,g_n, h).
 # Examples
 ```jldoctest
 julia> G = cyclic_group(3)
-Pc group of order 3
+Pc group of order 3 with 1 generator f1
 
 julia> H = symmetric_group(2)
-Sym(2)
+Symmetric group of degree 2 and order 2 with 1 generator
+  (1,2)
 
 julia> W = wreath_product(G,H)
-<group of size 18 with 2 generators>
+Wreath product of
+  pc group of order 3 wr symmetric group of degree 2 and order 2
```

Comment:
From a linguistic POV I would say that after `Wreath product of` there should either be a list of things or something like `A in B` or so, but not `A wr B`, which is already the complete notation for the wreath product